oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxCentralProcessor.java

```diff
@@ -299,11 +300,11 @@ private String getProcessorID(String stepping, String model, String family, Stri
             if (checkLine.contains(marker) && checkLine.trim().startsWith("0x00000001")) {
                 String eax = "";
                 String edx = "";
-                for (String register : checkLine.split("\\s+")) {
+                for (String register : ParseUtil.whitespace.split(checkLine)) {
                     if (register.startsWith("eax=")) {
-                        eax = register.replace("eax=0x", "");
+                        eax = StringUtils.replace(register, "eax=0x", "");

```

Comment:
I just added a new code snippet to my last answer, with a possible implementation of the new method